recipes/singlem/LICENSE

```diff
@@ -0,0 +1,675 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
Can you please request upstream that the license be included with the tarball? See https://bioconda.github.io/contributor/linting.html#gpl-requires-license-distributed

Reply:
Will do thanks for pointing that out.